### ISSUE TITLE
Question about PCGrad algorithm

### ISSUE THREAD

**Issue by MingChaoXu** [tianheyu927/PCGrad#18]
if there are different data sources in training data, for example, train task 1, 2 and 3, and task 1 only has labels of task 1. task 2 and 3 are also the same. can this algorithm optimize this question?

**Comment by sven-lijie** [comment]
In principle, no matter how multi-tasks are organized, as long as they act on the same layer of network during a gradient descent, this method can be applied.
